si/orange3-text.jaml

```diff
@@ -0,0 +1,5182 @@
+    class `OWAnnotator`:
+        Annotated Corpus Map: Anotirani zemljevid korpusa
+        Annotates projection clusters.: Anotira projekcijske gruče.
+        icons/Annotator.svg: false
+        annotated corpus map, annotator: annotated corpus map, annotator, anotator
+        class `Inputs`:
+            Corpus: Korpus
+        class `Outputs`:
+            Selected Docs: Izbrani dokumenti
+            Corpus: Korpus
+            Scores: Rezultati
+        class `Warning`:
+            'Selected features for Axis x ': 'Izbrane funkcije za os x '
+            and Axis y should differ.: in os y se morajo razlikovati.
+        class `Error`:
+            Data has no continuous variables.: Podatki nimajo številskih spremenljivk.
+            Not enough instances in data.: V podatkih ni dovolj primerov.
+            An error occurred while annotating data.\n{}: Pri anotiranju podatkov je prišlo do napake.\n{}
+        def `_add_controls`:
+            Cluster labels:: Oznake gruč:
+            n_cluster_labels: false
+            show_cluster_hull: false
+            Show cluster hull: Pokaži trup gruče
+            color_by_cluster: false
+            Color points by cluster: Pobarvaj točke glede na gručo
+        def `__add_annotation_controls`:
+            labelWidth: false
+            orientation: false
+            searchable: false
+            sendSelectedValue: false
+            contentsLength: false
+            Axes: Osi
+            attr_x: false
+            Axis x:: Os x:
+            attr_y: false
+            Axis y:: Os y:
+            Annotation: Oznake
+            clustering_type: false
+            DBSCAN: true
+            use_epsilon: false
+            epsilon:: true
+            epsilon: false
+            Gaussian mixture models: Gaussovi mešani modeli
+            use_n_components: false
+            clusters:: gruče:
+            n_components: false
+            From variable: Iz spremenljivk
+            cluster_var: false
+            p_threshold: false
+            FDR threshold:: FDR prag:
+        def `_run`:
+            No valid embedding data.: Ni veljavnih podatkov za vložitev.
+        def `send_scores`:
+            Scores: Rezultati
+            Score({label}): Rezultat({label})
+            p_value({label}): p-vrednost({label})
+        def `_get_projection_data`:
+            Cluster: Gruča
+        def `_get_send_report_caption`:
+            Clusters: Gruče
+            Color: Barva
+            Label: Oznaka
+            Shape: Oblika
+            Size: Velikost
+            Jittering: Tresenje
+            {} %: false
+    __main__: false
+    book-excerpts: false
+    \w+: false
+    English: false
+widgets/owbagofwords.py:
+    class `OWTBagOfWords`:
+        Bag of Words: Vreča besed
+        Generates a bag of words from the input corpus.: Iz vhodnega korpusa ustvari vrečo besed.
+        icons/BagOfWords.svg: false
+        bag of words, bow: bag of words, bow, besedilo
```

Comment:
Since `Vreča besed` is already a name of the widget Bag of words it can be ommited in the keywords of this widget. But you have a point with `besedilo`, so I will put it away

Reply:
Aaa, sure, it totally makes sense.